testsuite/tests/float-unboxing/float_subst_boxed_number.ml

```diff
@@ -139,6 +139,11 @@ let unbox_minor_words () =
     ignore (Gc.minor_words () = 0.)
   done
 
+let ignore_useless_args () =
+  let f x _y = int_of_float (cos x) in
```

Comment:
I've added `[@inlined always]` to the call site (the attribute on the definition would be `@@inline`, no?).